Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -532,8 +532,11 @@ def audit_conflicts
       end
     end
 
+    versioned_conflicts_whitelist = %w[node@ bash-completion@]
```

Comment:
Ok. I can change it to that, but the issue then is that it would need to be moved out method, and into the FormulaAuditor class scope to prevent a 'dynamic constant assignment' error. Also, there are other whitelists that aren't made into constants, https://github.com/Homebrew/brew/blob/master/Library/Homebrew/dev-cmd/audit.rb#L768-L790, for example. However, it may make sense to make it a true constant. I will create a comment on the current code so that it is more visible.